app/controllers/items_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ItemsController < ApplicationController
+  def index
+    @items = Item.all
+  end
+
+  def show
+    @item = Item.find(params[:id])
+  end
+
+  def new
+    @merchant = Merchant.find(params[:merchant_id])
+  end
+
+  def create
+    @merchant = Merchant.find(params[:merchant_id])
+    @item = @merchant.items.create!(item_params)
+    @item.status = true;
```

Comment:
Be careful with using `=` to set attributes. This will only change the temporary Ruby object, but won't actually make a change to the database. You have to call `@item.save` after if you want it to save in the DB.